models/src/main/java/com/vimeo/networking2/SubscriptionRenewal.kt

```diff
@@ -0,0 +1,19 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Information about the user's next renewal.
+ */
+@JsonClass(generateAdapter = true)
+data class SubscriptionRenewal(
+
+    /**
+     * The date in YYYY-MM-DD format when the user's membership renews (or expires, if they have
+     * disabled autorenew). For display only.
+     */
+    @Json(name = "display_date")
+    val displayDate: String
```

Comment:
This should be a date type.

Reply:
Based on the name and description of this field I'm thinking this should maybe stay a string...? It is apparently "for display only." Might also be an issue to actually use this field because it is formatted into a String by the backend PHP code so we lose timezone information meaning the data in the field could actually be incorrect.